.buildkite/scripts/integration-tests.ps1

```diff
@@ -22,7 +22,11 @@ $env:AGENT_VERSION = $PACKAGE_VERSION
 $env:SNAPSHOT = $true
 
 echo "~~~ Building test binaries"
-mage build:testBinaries
+& mage build:testBinaries
```

Comment:
Is `&` actually needed here? My understanding is that `&` is needed to run a command stored in a variable, so a direct execution would work.